wrapanapi/systems/ec2.py

```diff
@@ -1595,3 +1596,41 @@ def cleanup_resources(self):
         self.remove_all_unused_nics()
         self.remove_all_unused_volumes()
         self.remove_all_unused_ips()
+
+    def list_ocps(self):
+        """
+        List openshift clusters (name, metadata, resources) where metadata can be used for uninstalling the cluster
+        """
+        ocp_list = []
+        resources = self.resourcegroupstaggingapi_connection.get_resources()["ResourceTagMappingList"]
+        for resource in resources:
+            tags = resource["Tags"]
+            for tag in tags:
+                key = tag["Key"]
+                if key.startswith("kubernetes.io/cluster/"):
+                    ocp = None
+                    ocp_name = key[22:]
+                    for ocp_item in ocp_list:
+                        if ocp_item["name"] == ocp_name:
+                            ocp = ocp_item
+                            break
+                    if not ocp:
```

Comment:
Consider using for/else syntax here, removes the need to initalize `ocp` and you're already using a `break`.